controllers/users.controller.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const User = require('../models/user.model');
+const db = require('../db');
+const _ = require('lodash');
+
+module.exports.handleAuthorizeUser = async (ctx, next) => {
+  const user = await getFilteredUser(ctx.status.accessToken);
+  ctx.status = 200;
+  ctx.body = user;
+};
+
+
+module.exports.handleUserGet = async (ctx, next) => {
+  const user = await getFilteredUser(ctx.status.accessToken, filterAccessToken);
+  ctx.status = 200;
+  ctx.body = user;
```

Comment:
In this case, rely on `getFilteredUser` middleware as usual, and use a serializer to print your user.

`ctx.body = UserSerializer.serialize(user)`

A serializer is a function that takes a model and responds with secure-to-print object representing the instance.